src/CoreMidi/MidiServices.cs

```diff
@@ -836,13 +836,16 @@ protected virtual void Dispose (bool disposing)
 
 		public IntPtr Bytes {
 			get {
+				byte* value = byteptr;
 				if (bytes is null || bytes.Length < 1)
-					return byteptr;
+					return value;
+				
 				unsafe {
 					fixed (byte* p = &bytes [start]) {
-						return (IntPtr) p;
+						value =(IntPtr) p;
```

Comment:
This doesn't actually change anything, the property still return the same value.

I had a quick look, and it seems we don't use this property ourselves. This would indicate we could just remove the property, but it's a property we expose in our public API, so we can't do that right away.

However, we can do this:

1. Add an Obsolete attribute that tells the user that this property isn't safe, and to use either the `ByteArray` or the `BytePointer` property.
2. Make the `ByteArray` and `BytePointer` properties public.
3. Surround the `Bytes` property with `#if !XAMCORE_5_0 ... #endif` - the `XAMCORE_5_0` define will be set the next time we can break our public API, and this way we'll remove the unsafe property at that point.